src/components/SignUpForm/index.js

```diff
@@ -34,35 +39,76 @@ const SignUpForm = ({ onSubmit }) => {
   const inputProps = useTextInputProps(handleValueChange, handleBlur, values);
 
   return (
-    <>
+    <View style={styles.container}>
+      <Input
+        label={strings.SIGN_UP.name}
+        testID="name-input"
+        error={errors.name}
```

Comment:
You can do: 

```suggestion
        error={errors[FIELDS.password]}
```
and use the array where you define the fields